.github/workflows/conflict.yaml

```
@@ -104,7 +104,16 @@ jobs:
                                 // if PR has no conflicts, remove the label
                                 if (pullRequestData.data.labels.find(label => label.name === 'Has Conflicts')) {
                                     console.log(`Removing 'Has Conflicts' label from PR #${prNumber}`);
-                                    await removeLabel(['Has Conflicts'], prNumber);
+                                    try {
+                                        await removeLabel(['Has Conflicts'], prNumber);
+                                    } catch(err) {
+                                        if (err.status === 404) {
+                                            console.log("(Has Conflicts) label found missing when trying to remove it");
```

tiny grammar nitpick, so in form of committable suggestion

```suggestion
                                            console.log("(Has Conflicts) label no longer found when trying to remove it");
```